src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -183,3 +183,46 @@ func (w logglyErrorWriter) Write(b []byte) (int, error) {
 	}
 	return len(b), nil
 }
+
+type nonStopWriter struct {
+	writers []io.Writer
+}
+
+// NonStopWriter creates a writer that duplicates its writes to all the
+// provided writers, even if errors encountered while writting.
+func NonStopWriter(writers ...io.Writer) io.Writer {
+	w := make([]io.Writer, len(writers))
+	copy(w, writers)
```

Comment:
I just copied from code of [`io.MultiWriter`](http://golang.org/src/io/multi.go#L57), not clear why.